src/Symfony/Bundle/WebProfilerBundle/Resources/views/Script/Mermaid/Makefile

```diff
@@ -0,0 +1,26 @@
+define diagram-orchestration
+import { diagram as flowchartV2 } from '../diagrams/flowchart/flowDiagram-v2.js';
+import { registerDiagram } from './diagramAPI.js';
+
+let hasLoadedDiagrams = false;
+export const addDiagrams = () => {
+  if (hasLoadedDiagrams) {
+    return;
+  }
+  hasLoadedDiagrams = true;
+  registerDiagram('flowchart-v2', flowchartV2, () => true);
+};
+endef
+
+.PHONY: mermaid-flowchart-v2.min.js
+mermaid-flowchart-v2.min.js: | mermaid-$(if $(tag),$(tag),$(error missing “tag” variable))/node_modules
+	$(file >mermaid-$(tag)/packages/mermaid/src/diagram-api/diagram-orchestration.ts,$(diagram-orchestration))
+	pnpm -C mermaid-$(tag) run build
+	cp mermaid-$(tag)/packages/mermaid/dist/mermaid.min.js $@
+
+mermaid-%/node_modules: | mermaid-%
+	pnpm -C $(@D) install --ignore-scripts
+
+mermaid-%:
+	curl -fL https://github.com/mermaid-js/mermaid/archive/refs/tags/$*.tar.gz | tar -xz --strip-components=1 --one-top-level=$@
```

Comment:
this looks like it will create a folder containing the mermaid source code. This should have a gitignore rule to avoid committing it by mistake.

Reply:
Okay, was not sure if I was supposed to add a `clean` rule. Will update `.gitignore`.

Reply:
we can also have a clean command, but that does not remove the need for a gitignore to prevent mistakes.